server/src/main/java/org/opensearch/search/startree/filter/ExactMatchDimFilter.java

```diff
@@ -49,8 +49,9 @@ public void initialiseForSegment(StarTreeValues starTreeValues, SearchContext se
             dimensionName,
             starTreeValues.getStarTreeField().getDimensionsOrder()
         );
-        this.dimensionFilterMapper = DimensionFilterMapper.Factory.fromMappedFieldType(
-            searchContext.mapperService().fieldType(dimensionName)
+        DimensionFilterMapper dimensionFilterMapper = DimensionFilterMapper.Factory.fromMappedFieldType(
+            searchContext.mapperService().fieldType(dimensionName),
+            searchContext
```

Comment:
With this change, can we remove the field on line 38?

Reply:
This got crept in by mistake. I'm reverting it back to reference class variable only to keep it consistent with `RangeMatchDimFilter`